### PR TITLE
System.Tuple<T1,T2,T3,T4,T5,T6,T7,TRest> F# snippets

### PR DESCRIPTION
**Pull request by albert-du**
Adds F# snippets for Tuple<T1,T2,T3,T4,T5,T6,T7,TRest>

### CODE REVIEW

**Review by BillWagner — APPROVED** [review]
Thanks again @albert-du 

I've reviewed these changes, and I'll :shipit: now.

File: snippets/fsharp/System/TupleT1,T2,T3,T4,T5,T6,T7,TRest/Item1/item1.fs [hunk 1]
```diff
@@ -0,0 +1,67 @@
+﻿open System
```
Comment (gewarren) [inline snippets/fsharp/System/TupleT1,T2,T3,T4,T5,T6,T7,TRest/Item1/item1.fs:1]: It looks like you forgot the `// <Snippet1>` tags in the various Item snippets. That's what caused the build warnings. @albert-du @BillWagner

Comment (BillWagner) [reply snippets/fsharp/System/TupleT1,T2,T3,T4,T5,T6,T7,TRest/Item1/item1.fs:1]: @albert-du Can you make an updated PR, or should I?

Comment (albert-du) [reply snippets/fsharp/System/TupleT1,T2,T3,T4,T5,T6,T7,TRest/Item1/item1.fs:1]: oops looks like this one fell off my radar a bit, thank you for fixing it!

**Pull request merged**